epuck_ros2_camera/camera_info/camera.yaml

```diff
@@ -0,0 +1,28 @@
+image_width: 640
+image_height: 480
+camera_name: camera
+camera_matrix:
+  rows: 3
+  cols: 3
+  data: [322.0704122808738, 0, 320.0, 0, 320.8673986158544, 240.0, 0, 0, 1]
```

Comment:
Ho did you estimate these (and the following ones) coefficient?

Reply:
The principal point is from is based on camera resolution, `cx` and `cy` should represent a point near the image centre. The focal length is taken from the calibrated Raspberry Pi camera.

Reply:
Let me try to do a better estimation...

Reply:
I was not saying these numbers are wrong, I was just trying to understand from where they come from.

Reply:
Calibrated. Focal length was very inaccurate.

Reply:
> I was not saying these numbers are wrong, I was just trying to understand from where they come from.

I wanted to calibrate it anyway :)